plans/libev/plan.sh

```diff
@@ -0,0 +1,12 @@
+pkg_origin=core
+pkg_name=libev
+pkg_version=4.22
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh"
+pkg_license=('MIT')
```

Comment:
the license appears to be `BSD-2-Clause` and `GPL-2.0`:
http://cvs.schmorp.de/libev/LICENSE?revision=1.11&view=markup